numpy/core/src/multiarray/nditer_pywrap.c

```diff
@@ -20,12 +20,16 @@
 
 typedef struct NewNpyArrayIterObject_tag NewNpyArrayIterObject;
 
+enum NPYITER_CONTEXT {CONTEXT_NOTENTERED, CONTEXT_INSIDE, CONTEXT_EXITED};
+
 struct NewNpyArrayIterObject_tag {
     PyObject_HEAD
     /* The iterator */
     NpyIter *iter;
     /* Flag indicating iteration started/stopped */
     char started, finished;
+    /* iter must used as a context manager if writebackifcopy semantics used */
+    char managed;
```

Comment:
The comment above this line is no longer correct, right?

Reply:
fixed